io/teca_cf_reader.cxx

```diff
@@ -287,7 +286,7 @@ class read_variable
             // success!
             return std::make_pair(m_id, var);
             )
-
+        
```

Comment:
sorry to nitpick, but you added a line of spaces here rather than an empty line. does sublime support highlighting that? vim does, it makes it easy to catch.

Reply:
Interesting! nice catch. another reason to use vim :)